modules/op-resolver-git-scm.adoc

```diff
@@ -99,38 +100,43 @@ metadata:
   name: pipeline-with-git-task-reference-demo
 spec:
   tasks:
-  - name: "git-task-reference-demo"
-    taskRef:
-      resolver: git
-      params:
-      - name: url
-        value: https://github.com/tektoncd/catalog.git
-      - name: revision
-        value: main
-      - name: pathInRepo
-        value: task/git-clone/0.6/git-clone.yaml
-      - name: sample-task-parameter
-        value: test
+    - name: "git-task-reference-demo"
+      taskRef:
+        resolver: git
+        params:
+          - name: repo
+            value: catalog
+          - name: org
+            value: tektoncd
+          - name: revision
+            value: main
+          - name: pathInRepo
+            value: task/git-clone/0.6/git-clone.yaml
+          - name: sample-task-parameter
+            value: test
```

Comment:
Same here, why are we putting this "random" parameter here ? It might render the request invalid.